other-docs/guides/upgrading/v14.md

```diff
@@ -89,6 +89,19 @@ Altis Analytics also used the Workflows functionality to provide notifications
 of Experiment results and so on. If you wish to reproduce that behaviour, you
 can use the example in this [GitHub Gist](https://gist.github.com/mikelittle/f73ce199705d85f9a91431e9b88ad60a).
 
+#### Clone and Amend functionality
+
+Altis v14 no longer provides Yoast Duplicate Post plugin that provided "Clone" and "Amend" functionality. However, you
+can install the off-the-shelf plugin in your project's `composer.json` file.
+
+```shell
+## Add the Yoast Duplicate Post plugin
+composer require "wppackagist/duplicate-post"
+```
+
+Note: The plugin uses the terms "Clone" and "Copy to a new draft" instead of "Clone" and "Rewrite & Republish" instead of
```

Comment:
@mikelittle I think it would be much easier to read this as "Clone and Copy to a new draft" rather than "Clone" and "Copy to a new draft"

Reply:
Weirdly, "Clone" and "Copy to a new draft" are two similar but separate functions. Whereas Altis only had "Clone". So, although it reads awkwardly, it is accurate!